api/contrib/envoy/extensions/tap_sinks/udp_sink/v3alpha/udp_sink.proto

```diff
@@ -19,4 +21,14 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message UdpSink {
   // Configure UDP Address.
   config.core.v3.SocketAddress udp_address = 1;
+
+  // Configure UDP client send buffer size
+  google.protobuf.UInt32Value udp_sent_buffer_bytes = 2;
+
+  // Don't consider receive buffer size because it doesn't get any message from server
+
+  // Configure UDP max sending message size in each UDP transmission.
+  // This setting allows users to set the maximum size sent based on system-specific constraints.
+  // If not set explicitly, the default value is 63,488 bytes (based on linux 65507 bytes).
+  google.protobuf.UInt32Value udp_max_send_msg_size_bytes = 3;
```

Comment:
These seem to be core UDP connection properties (not only related to sinks). Would it make sense to declare them as core and use that data-type? The closest thing I'm aware of is [UdpSocketConfig](https://github.com/envoyproxy/envoy/blob/main/api/envoy/config/core/v3/udp_socket_config.proto), but I'm not sure if this is only being used on the listener part.